porn.txt

```diff
@@ -484624,6 +484628,8 @@
 0.0.0.0 xhamester.com
 0.0.0.0 xhamester.org
 0.0.0.0 xhammer.com
+0.0.0.0 xhammer.net
+0.0.0.0 xhammer.be
```

Comment:
redirect to https://xhammer.azurewebsites.net/

<details><summary>Logger output</summary>

| | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/favicon.ico` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_back.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentMain.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_5.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_6.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_7.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_4.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_8.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_3.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_2.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_1.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentBottom.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/website.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/community.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/events.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/battlenights.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/xhammer.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentTop.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationLogon.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationForum.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationLocation.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationHome.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationStrip.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | css | `https://xhammer.azurewebsites.net/css/xhammer.css` |
| +2 | *$document,permissions=browsing-topics=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=idle-detection=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=join-ad-interest-group=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=run-ad-auction=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=attribution-reporting=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | no-remote-fonts: * true | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | no-remote-fonts: * true | -- | xhammer.azurewebsites.net | 1 | get | inline-font | `https://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.azurewebsites.net | 1 | get | doc | `http://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.be | 1 | get | image | `http://xhammer.be/favicon.ico` |
| +1 | *$document,permissions=browsing-topics=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=idle-detection=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=join-ad-interest-group=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=run-ad-auction=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=attribution-reporting=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | no-remote-fonts: * true | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | no-remote-fonts: * true | -- | xhammer.be | 1 | get | inline-font | `http://xhammer.be/` |
| +1 |  |  | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +0 |  |  | xhammer.be | 1 | get | doc | `https://xhammer.be/` |
| +0 |  |  | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
</details>

Reply:
So what?